### DISCUSSION TITLE
Issue starting Polypheny-UI: "There is an issue with node-fibers"

### DISCUSSION THREAD

**Discussion by HashmiAS** [polypheny/Polypheny-DB#320] [Q&A] [answered]
Polypheny UI is not launching completely. While the website appears to launch on localhost the browser displays a message "CANNOT GET". The error in the terminal is "ISSUE WITH NODE FIBERS". Is there a command to reset polypheny UI like how reset catalog works in DB?

**Comment by vogti** [comment]
I never had this issue before. I found [this](https://stackoverflow.com/questions/48923395/there-is-an-issue-with-node-fibers/50685180) on Stack Overflow. Have you already tried the solutions suggested there?

**Comment by HashmiAS** [comment, marked as answer]
Yes seems it's an internal error of MACOS-Catalina. Reinstalling Xcode seems to be working for me.
